couchdb/values.yaml

```diff
@@ -95,6 +95,17 @@ affinity: {}
   #             - couchdb
   #       topologyKey: "kubernetes.io/hostname"
 
+## To add restrictions in topology spread constraints such as distribute pods across aws zones 
+## in eks aws, use podTopologySpreadConstraints
```

Comment:
Is this note about EKS saying that `topologySpreadConstraints` are ignored there? I can't find any reference to that effect.

Reply:
No, that was a confusing comment. I corrected it to avoid confusion.